app/main.py

```diff
@@ -27,28 +28,42 @@ def load_data(file_path: str) -> list[dict]:
 
 
 def video_with_highest_views(videos) -> str:
-    pass
+    videos.sort(key=lambda vide0: vide0['views'], reverse=True)
+    return videos[0]['title']
```

Comment:
sort is changing your list, so for all it will be changed, not a good point.
Also it can be one liner, and used max(), because you need to know only highest views, not to sort a list